posts/inside-rust/2020-07-24-opening-up-the-core-team-agenda.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Opening up the Core Team agenda"
+author: Pietro Albini
+team: the Core Team <https://www.rust-lang.org/governance/teams/core>
```

Comment:
Historically we have used

    The Rust Core Team

I like the addition of the link, but maybe for consistency's sake we should make the text the same.

Reply:
@steveklabnik note that this will show up as "Pietro Albini, on behalf of the [Core Team]", as any other Inside Rust post. Do we want to change that?

Reply:
Ah, I did not see that on Inside Rust! Seems fine :)